### PR TITLE
Update line numbers of cardano-serialisation-lib

### PR DESCRIPTION
**Pull request by thisHermit**

### CODE REVIEW

**Review by rphair — APPROVED** [review]
Looks accurate.  The current links on CIP-0008 don't point anywhere in particular in that file.

So approving as is but @SebastienGllmt maybe there's a more general way to refer to these code segments that will still work even if the code shifts around some more?

**Comment by thisHermit** [comment]
I see a way to point to a particular commit. Would that be an acceptable improvement?

**Comment by SebastienGllmt** [comment]
Github allows generating permalinks for files which will be more stable than the change you made

**Comment by rphair** [comment]
I only know of linking to a specific commit, which will also also change (along with the line number) when the code is updated.  What's the secret, @SebastienGllmt ... or is it best just to link to the file itself?

**Comment by rphair** [comment]
never mind that last comment... yes @thisHermit I think linking to the commit & line number be fine since you're only trying to show that section of code (I was just hoping someone might know how to search for some other type of anchor in the file, e.g a regex)

File: CIP-0008/README.md [hunk 1]
```diff
@@ -121,8 +121,8 @@ Other blockchains have existing specifications for message signing, but they mos
 
 Cardano already allows message signing within the WASM bindings. Notably,
 
-1) [sign](https://github.com/Emurgo/cardano-serialization-lib/blob/master/rust/src/crypto.rs#L260)
-2) [verify](https://github.com/Emurgo/cardano-serialization-lib/blob/master/rust/src/crypto.rs#L303)
+1) [sign](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L279)
+2) [verify](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L322)
 
 You can see an example of these two functions [here](https://repl.it/repls/FlusteredSimpleFunction)
```
[the remark this answers is not in the document]
Comment (SebastienGllmt) [reply CIP-0008/README.md:127]: no I don't know any good alternative

**Review by SebastienGllmt — APPROVED** [review]

**Review by KtorZ — APPROVED** [review]
I think the issue with repl.it is orthogonal to that PR. So, happy to merge this change now, and perhaps move the repl.it discussion to github [issues](https://github.com/cardano-foundation/CIPs/issues)

**Pull request merged**